### PR TITLE
Better graphics

### PR DESCRIPTION
**Pull request by takkaria**
This refactors and improves both the house and the 'target bar' display.  Both are now generated entirely using JavaScript template strings that substitue for SVG, with no external files.  The target bars are vector instead of canvas, and various other formatting issues have been resolved, e.g. the house's arrows' locations have been tweaked to avoid overlaps and the text is better aligned.

### CODE REVIEW

File: mhep/mhep/dev/static/dev/js/ui-mhep.js [hunk 1]
```diff
@@ -1,110 +1,12 @@
...
-        units: units,
-        targets: {
-            "70% heating saving": datasets.target_values.energy_use_per_person,
-            "UK Average": datasets.uk_average_values.energy_use_per_person
-        }
-    };
-    targetbar(parent + " #perperson", options);
-    // ---------------------------------------------------------------------------------
+    draw_space_heating_targetbar(parent, data)
+    draw_primary_energy_targetbar(parent, data
```
Comment (idrysdale) [inline mhep/mhep/dev/static/dev/js/ui-mhep.js]: This is missing the trailing `)`

**Comment by idrysdale** [comment]
This code is looking much better, nice one.

Running it locally, the roof and back wall of the house graphic are not displayed (nor does there appear to be svg code to draw them).

Screenshot 2019-11-25 at 09 13 45

Am I missing something?

**Comment by takkaria** [comment]
That is very weird.  I must have broken someting when moving these changes from v1 to dev - which I did by making patch files, search and replacing v1 with dev, and then applying the patch... which leaves some scope for errors to creep in.  Really surprised I didn't notice the house issue myself though!

**Comment by takkaria** [comment]
OK this should be good to go now, changes incorporated as fix-ups.  It was a case of s/v1/dev/ wrongly editing some SVG path data.

**Comment by idrysdale** [comment]
Great, nice work!

**Pull request merged**